.jenkins/pytorch/win-test.sh

```diff
@@ -12,156 +12,41 @@ fi
 
 export TMP_DIR="${PWD}/build/win_tmp"
 export TMP_DIR_WIN=$(cygpath -w "${TMP_DIR}")
-mkdir -p $TMP_DIR/ci_scripts/
+
+CI_SCRIPTS_DIR=$TMP_DIR/ci_scripts
```

Comment:
nit: export this variable, otherwise it won't be passed to the batch scripts

Reply:
This variable was intentionally local to only this script.